<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SourceButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>选择源文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TargetButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>选择目标文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Process">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>运行</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="SourceFolder">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TargetFolder">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ProcessText">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
